exercises/algorithms/src/main/java/com/cbfacademy/search/Search.java

```diff
@@ -0,0 +1,17 @@
+package com.cbfacademy.search;
+
+public interface Search {
+    /**
+     * Searches an array.
+     *
+     * @param sortedArray The array to be searched (the array should already be s
+     *                    rted)
+     * @param target      The value to be searched for in the array
+     */
+    int search(int[] sortedArray, int target);
+
+    /**
+     * Returns the exercise name
+     */
+    String getName();
```

Comment:
If there's no valid answer to offer a student who asks why this is present, I'd suggest we should remove any existing instances rather than adding more.